sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/RecognizePiiEntityAsyncClient.java

```diff
@@ -82,6 +82,7 @@
                             throw logger.logExceptionAsError(toTextAnalyticsException(entitiesResult.getError()));
```

Comment:
Should we also add this change to the changelog?

Reply:
? what changes?